src/layouts/PrivateAreaLayout/PrivateAreaLayout.jsx

```diff
@@ -56,7 +56,7 @@ const PrivateAreaLayout = (props) => {
   useInterval(updateSession, minToMillisec(60), true);
 
   const loadUserData = () => {
-    dispatch(getUserData(storeSession.tradeApi.accessToken));
+    dispatch(getUserData(storeSession.tradeApi.accessToken, true));
```

Comment:
@muradmalik777 Do we still need that? I think at some point we talked about duplicate requests. I'm not sure if I need to load the exchanges by passing `true` there.

Reply:
I was thinking maybe we should remove dailyBalance from redux, I don't see any special use case for storing in redux and this will be automatically fixed with that. I will have to check deeply but I can we remove from redux and use a hook.
about what you have commented, I think we only need to call with initial load, and here it should be "false". Things are getting mixed now!

Reply:
Better leave it like this and we dedicate some time to check everything, find code that can be refactored.

Reply:
Yeah but we talked about it months ago, at some point, we should really try to improve it step by step. Ok, thanks. I don't think we need to store it in redux either.